psi4/src/psi4/libmints/matrix.h

```diff
@@ -36,6 +36,8 @@
 
 #include "dimension.h"
 
+#include <eigen3/Eigen/Core>
+
```

Comment:
Isn't this a new requirement? I don't think Eigen3 has been used in Psi4 previously...

Reply:
Right, I did mean to mention this. Correct that Psi4 doesn't use Eigen3 itself. But Eigen3 and boost header-only libs are needed to compile against L2 header-only, so it isn't a net new requirement.